.eslintrc.js

```diff
@@ -140,6 +140,8 @@ module.exports = {
         'app/scripts/lib/**/*.test.js',
         'app/scripts/migrations/*.test.js',
         'app/scripts/platforms/*.test.js',
+        'app/scripts/lib/**/*.test.js',
```

Comment:
Looks like this was accidentally duplicated when resolving conflicts. Maybe because this list was re-ordered to be alphabetical (it's listed above now)

Reply:
😱 :

Reply:
Resolved in 69af06c